src/pages/home/report/ReportActionCompose/ReportActionCompose.tsx

```diff
@@ -323,7 +325,7 @@ function ReportActionCompose({
     // We are returning a callback here as we want to incoke the method on unmount only
     useEffect(
         () => () => {
-            if (!EmojiPickerActions.isActive(report?.reportID ?? '-1')) {
```

Comment:
Also here why are we removing the default `-1` value? Asking because I remember that historically we've been using reportID `-1` for some cases, and just wanted to make sure we're not introducing a regression here 😄

Reply:
Here you are: https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#default-value-for-inexistent-ids

We recently changed the default fallback value for inexistent IDs

Reply:
Oh cool, thanks for the clarification!